src/commcare_cloud/commcare_cloud.py

```diff
@@ -171,12 +171,25 @@ def make_command_parser(available_envs, formatter_class=RawTextHelpFormatter,
 
 
 def call_commcare_cloud(input_argv=sys.argv):
+    # make sure user aware they are running on python 2 env
+    force_python_2 = "--force-commcare-cloud-to-use-python2"
+    if not os.environ.get("TRAVIS_TEST") and sys.version_info[0] == 2:
+        if force_python_2 not in input_argv:
+            print('Error: you must upgrade to Python 3. Though not desirable, if you really have to '
+                  'you can use Python 2 with this option {}'.format(force_python_2))
```

Comment:
We should provide instructions for creating a Python 3 virtualenv in this error message. That will probably be more effective than simply providing an option to continue with Python 2.
```suggestion
            from textwrap import dedent
            print(dedent(f"""
                Error: you must upgrade to Python 3. Though not desirable, if you really have
                to you can use Python 2 with this option {force_python_2}

                Setup Python 3
                - Create a new Python-3-based virtualenv
                - pip install -r requirements3.txt
                - rm -rf src/commcare_cloud.egg-info
                - pip install -e .
                - manage-commcare-cloud install
            """))
```

Reply:
Made the change but stuck with `"{}".format()` since this needs to work on Python 2